src/application.coffee

```diff
@@ -103,8 +103,12 @@ logSpecialAction = (action, value, success) ->
 	logSystemMessage(msg, {}, "Apply special action #{if success then "success" else "in progress"}")
 
 application = {}
+application.UpdatesLockedError = UpdatesLockedError
 
-application.kill = kill = (app, updateDB = true, removeContainer = true) ->
+application.kill = kill = (app, opts = {}) ->
+	{ updateDB, removeContainer } = opts
```

Comment:
Just realized this doesn't work on CoffeeScript 1.9. We need to upgrade it, but it should be done separately.